syft/generic/pointers/multi_pointer.py

```diff
@@ -209,6 +210,7 @@ def handle_func_command(cls, command):
             results[worker] = new_type.handle_func_command(new_command)
 
         # Put back MultiPointerTensor on the tensors found in the response
+        # TODO[yanndupis]: get rid of these torch references when extending hook_args
```

Comment:
Should this be done (or should an issue be made for this)

Reply:
it's the same TODO as above. I think this generalization work should be done before we open this line of work for contributions, so maybe best to hold off on an issue

Reply:
https://github.com/OpenMined/PySyft/issues/2530

Reply:
can you link it in the code